src/modules/dashboard/components/ActionsPage/StakingWidget/StakingWidget.tsx

```diff
@@ -59,6 +66,8 @@ const validationSchema = yup.object({
   amount: yup.number(),
 });
 
+const limit = 4000;
```

Comment:
I usually like to have variables like this in capital letters

Reply:
CharacterCount.configure() requires limit to be lower case.

Reply:
> CharacterCount.configure() requires limit to be lower case. This comes from the tiptap extension.

It requires the property name (key in the object) to be lower case but it doesn't oblige us to name our variables in any way. It can be any name:
`CharacterCount.configure({ limit: LIMIT })

Reply:
I see. Thank you for pointing this out.